docs/user-manual.md

```diff
@@ -0,0 +1,80 @@
+
+After each stereoshift measurement, the tool will remember position of the fiducial markings and the POI. This is useful if you need to measure the stereoshift for different points in the same region of the bubble chamber.
+
+### Measuring the image magnification
+In addition to the properties associated with a specific particle, the tool allows you to measure the image magnification. As explained in the lab manual[^3], this is done by measuring the projected distance between two pairs of fiducial markings, one at the top and one at the bottom of the bubble chamber. To do this, click on the `Measure magnification` button. This will enable the magnification tool, which will ask you to select and identify four fiducial positions in the image. The tool will then calculate the magnification parameters, which, combined with a measurement of the depth, can be used to convert the measurements of the particle properties to real units.
+
+Once computed for the first time, the magnification parameters are stored and used to convert all measurements. If you need to recompute the magnification parameters, you can do so by clicking on the `Measure magnification` button again. The tool will remember previously computed magnification parameters, and will allow you to switch between them.
+
+### Saving the data
+To save the data, click on the `Save to file` button. This will open a file dialog, which will allow you to select the file where you want to save the data. The tool will save the data in a CSV file, which will contain the information about the particles, and their properties, as well as the magnification parameters.
+
+## Useful keyboard shortcuts
+Some tips...
+
+## References
+[^1]:For tests you can use the data stored in `/r01/ParticleTracks/real_data` on the HEP machines.
+
+[^2]: Showing particle ID and the frame where the particle is located.
+
+[^3]: Particle Tracks lab manual, 2024.
+
+[^4]: Realised that for the measurement of the $\Lambda^0$ students are asked to measure the stereoshift for both the origin and decay vertex, to obtain the total decay lenth. At the moment this is not possible with the tool, as it only allows to measure the stereoshift for a single point. Need to think how to implement this, or work around it.
+
+[^5]: Understand if the Back/Front option is now obsolete.
+
+[napari]: https://napari.org/stable/
+[stereoshift]: https://www.hep.phy.cam.ac.uk/~palvarez/ParticleTracks/
+[def]: #stereoshift
```

Comment:
```suggestion
```

Is this line needed?

Reply:
I dont think so, unsure where it came from...